Modules/Core/Common/include/itkConstNeighborhoodIterator.hxx

```diff
@@ -568,9 +568,7 @@ ConstNeighborhoodIterator<TImage, TBoundaryCondition>::PrintSelf(std::ostream &
 {
   DimensionValueType i;
 
-  os << indent;
-  os << "ConstNeighborhoodIterator {this= " << this;
-  os << ", m_Region = { Start = {";
+  os << indent << "ConstNeighborhoodIterator {this= " << this << ", m_Region = { Start = {";
```

Comment:
I like this!

Reply:
Your force-push removed this change :(

Reply:
@N-Dekker shall I merge this PR or are there some details that still need another round of force pushes?

Reply:
@jhlegarreta Thanks for asking. I think the PR is ready, although it has excluded output streams named `os`, as those are mostly from PrintSelf implementations. PrintSelf may be dealt with in a different way, in another PR, based on the discussion I had with Dženan (@dzenanz), starting at https://github.com/InsightSoftwareConsortium/ITK/pull/3802#discussion_r1051069125

Moreover, I'm not really sure if Dženan likes the idea of this PR. I mean, I'm proposing to adjust the following code:

    std::cout << line1 << std::endl;
    std::cout << line2 << std::endl;
  
Replacing these two statements with a single statement:

    std::cout << line1 << std::endl
              << line2 << std::endl;

I like it, because it does not unnecessarily repeat the name of the output stream. However, other developers may not like it, I don't know. 

It's fine to me if Dženan would like to wait a little longer, to see what other people think. Anyway, thanks for your support, Jon.

Reply:
OK. I'll merge it, but note that
- Unless this is automated, we will not be able to enforce this automatically. A script to do this automatically would be best.
- Please, go through the ITK SWG and do the corresponding changes to the coding style section. Also it is highly likely that the ITK SWG `Examples` were not included in this PR, so please do them in a separate PR.